governance/votes/Executive vote - September 13, 2023.md

```diff
@@ -0,0 +1,174 @@
+- The Spark Protocol Dai Direct Deposit Module (D3M) Maximum Debt Ceiling will be increased to 400 million DAI; and the Ceiling Increase Cooldown will be increased to 12 hours. 
+- The Fortunafi vault's Debt Ceiling will be set to zero DAI, per the Stability Scope Artifact's requirement to offboard legacy Legal Recourse Assets.
+- A DAO resolution pertaining to the HV Bank vault will be approved to terminate future purchases and return available cash from the HVB Master Participation Trust. This is per the Stability Scope Artifact's requirement to offboard legacy Legal Recourse Assets.
+- Aligned Delegate compensation for August 2023 will be distributed.
+- A Spark Protocol Proxy Spell will be triggered that implements changes to the DAI borrow spread and the flash loan fee.
+- MKR vesting transfers will be executed for the DECO-001 and SES-001 Core Units, per their successful budget proposals. 
+- The `MCD_CAT` contract will be scuttled by removing it from the Chainlog and removing all its permissions on the Protocol contracts.  
+
+**Voting for this executive proposal will place your MKR in support of the actions outlined above.**
+
+Unless otherwise noted, the actions listed above are subject to the [GSM Pause Delay](https://manual.makerdao.com/parameter-index/core/param-gsm-pause-delay). This means that if this executive proposal passes, the actions listed above will only become active in the Maker Protocol after the GSM Pause Delay has expired. The GSM Pause Delay is currently set to **48 hours**.
+
+If this executive proposal does not pass within 30 days, then it will expire and can no longer have any effect on the Maker Protocol.
+
+
+---
+
+## Proposal Details
+
+For voters' convenience, each proposed item is prefaced with information showing its provenance. 
+
+- "Authorization" links to the official document(s) or governance action authorizing the proposed item's inclusion in the Executive Vote. This can be the Atlas, a Scope Alignment Artifact, a successful governance poll and/or a ratified budget proposal. 
+- "Reasoning" links to the MakerDAO Forum post providing the rationale and additional context for each item. 
+- "Authorizing Parties" may also be listed (with or without links). These may include the responsible Scope Facilitator and/or Scope Advisory Council members.
+
+### Stability-Scope Defined Parameter Changes
+
+*Authorization*: [Stability Scope Artifact](https://mips.makerdao.com/mips/details/MIP104#0-the-stability-scope)\
+*Reasoning*: [Forum post](https://forum.makerdao.com/t/stability-scope-parameter-changes-5/21969/1) "Stability Scope Parameter Changes"\
+*Authorizing Party:* [Stability Scope Facilitator](http://forum.makerdao.com/t/stability-scope-parameter-changes-5/21969/4)
+
+The Scope-defined parameter changes below have been proposed by Stability Scope Advisory Council Member [Block Analitica (BA Labs)](https://forum.makerdao.com/t/professional-ecosystem-actor-introduction-ba-labs/20813): 
```

Comment:
why not just add a "*Proposal*:" section along with the other things you've selected above, will reduce word count